geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ServerConnection.java

```diff
@@ -1886,12 +1826,49 @@ private long getUniqueId(Principal principal) throws Exception {
           securityLogWriter.warning(
               String.format(
                   "%s: Post-process authorization enabled, but no authentication callback (%s) is configured",
-                  new Object[] {getName(), SECURITY_CLIENT_AUTHENTICATOR}));
+                  getName(), SECURITY_CLIENT_AUTHENTICATOR));
         }
       }
       postAuthzRequest =
           new AuthorizeRequestPP(postAuthzFactoryName, getProxyID(), principal, getCache());
     }
     return setUserAuthorizeAndPostAuthorizeRequest(authzRequest, postAuthzRequest);
   }
+
+  static class ProcessingMessageTimer {
+    public static final long NOT_PROCESSING = -1L;
```

Comment:
It looks like this is only needed by this class and the unit test. If you remove the "public" can the unit test still see it? If so I think the "public" should be removed.

Reply:
Unit test uses it, so I can make it `package private`.